docs/user-interface/graphics/colors.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Colors"
+description: "The .NET MAUI Color class, in the Microsoft.Maui.Graphics namespace, lets you specify colors as RGB values, HSL values, HSV values, or with a color name."
+ms.date: 08/31/2021
+---
+
+# Colors
+
+<!-- Sample link goes here -->
+
+The `Color` class, in the `Microsoft.Maui.Graphics` namespace, lets you specify colors as Red-Green-Blue (RGB) values, Hue-Saturation-Luminosity (HSL) values, Hue-Saturation-Value (HSV) values, or with a color name. An Alpha channel is also available to indicate transparency.
+
+`Color` objects can be created with `Color` constructors, which can be used to specify a gray shade, an RGB value, or an RGB value with transparency. In all cases, arguments are `float` values ranging from 0 to 1.
+
+> [!NOTE]
+> The default `Color` constructor creates a black `Color` object.
+
+You can also use the following static methods to create `Color` objects:
+
+- `Color.FromRgb` from `float` RGB values that range from 0 to 1.
+- `Color.FromRgb` from `double` RGB values that range from 0 to 1.
+- `Color.FromRgb` from `byte` RGB values that range from 0 to 255.
+- `Color.FromRgba` from `float` RGBA values that range from 0 to 1.
+- `Color.FromRgba` from `double` RGBA values that range from 0 to 1.
+- `Color.FromRgba` from `byte` RGBA values that range from 0 to 255.
+- `Color.FromRgba` from a `string`-based hexadecimal value in the form "#RRGGBBAA" or "#RRGGBB" or "#RGBA" or "#RGB", where each letter corresponds to a hexadecimal digit for the alpha, red, green, and blue channels.
+- `Color.FromHsla` from `float` HSLA values.
+- `Color.FromHsla` from `double` HSLA values.
+- `Color.FromHsv` from `float` HSV values that range from 0 to 1.
+- `Color.FromHsv` from `int` HSV values that range from 0 to 255.
+- `Color.FromHsva` from `float` HSVA values.
+- `Color.FromHsva` from `int` HSV values.
+- `Color.FromInt` from an `int` value calculated as (B + 256 \* (G + 256 \* (R + 256 \* A))).
+- `Color.FromUint` from a `uint` value calculated as (B + 256 \* (G + 256 \* (R + 256 \* A))).
+- `Color.FromArgb` from a `string`-based hexadecimal value in the form "#AARRGGBB" or "#RRGGBB" or "#ARGB" or "RGB", where each letter corresponds to a hexadecimal digit for the alpha, red, green, and blue channels.
```

Comment:
I don't know if it is worth highlighting, but whoever is migrating from Xamarin.Forms should use this method to replace `Color.FromHex`

Reply:
I won't highlight this here, but there will be a migration doc that lists all the changes Xamarin.Forms users need to know about.